requirements.txt

```diff
@@ -1,5 +1,6 @@
 certifi
 h11
+h2
```

Comment:
We also need to add it to `install_requires` in the setup.py

Reply:
Good catch - thanks!